velobike/history.go

```diff
@@ -8,36 +10,36 @@ type HistoryService struct {
 // History describes the body of ride/history method response.
 type History struct {
 	Items          []HistoryItem `json:"Items,omitempty"`
-	TotalRidesTime *string       `json:"TotalRidesTime,omitempty"`
+	TotalRidesTime time.Time     `json:"TotalRidesTime,omitempty"`
```

Comment:
From my point of view, there is some sense to use pointers (as it was implemented for strings) -- if the field is not presented it in the response, it's better to have it as a nil pointer, not as a zero value for `time.Time`. What do you think?